systemd/resolved/config.sls

```diff
@@ -0,0 +1,39 @@
+{%- from "systemd/map.jinja" import systemd with context -%}
```

Comment:
Perhaps leave off the extra whitespace control so only `{%- ... %}` (removing the second `-`).